src/components/navigation-header/navigation-header.module.css

```diff
@@ -111,6 +111,12 @@
 		border-color: var(--token-color-palette-neutral-200);
 	}
 
+	@nest html[data-theme='light'] & {
+		&:active {
+			background-color: var(--token-color-foreground-primary);
+		}
+	}
```

Comment:
This only applies to the activator classname for the user dropdown disclosure component, and won't affect other dropdown disclosure activator states.